examples/nbody/nbody.cpp

```diff
@@ -303,56 +303,56 @@ struct MoveKernel
     >
     LLAMA_FN_HOST_ACC_INLINE
     void operator()(
-        T_Acc const &acc,
+        T_Acc const & acc,
         T_View particles,
         Element ts
     ) const
     {
-        auto threadIndex  = alpaka::idx::getIdx<
+        auto threadIndex = alpaka::idx::getIdx<
             alpaka::Grid,
             alpaka::Threads
         >( acc )[ 0u ];
 
         auto const start = threadIndex * elems;
         auto const   end = alpaka::math::min(
             acc,
-            (threadIndex + 1) * elems,
+            ( threadIndex + 1 ) * elems,
             problemSize
         );
 
         LLAMA_INDEPENDENT_DATA
         for ( auto pos = start; pos < end; ++pos )
-            particles( pos )( dd::Pos() ) += particles( pos )( dd::Vel() ) * ts;
+            particles( pos )( dd::Pos( ) ) += particles( pos )( dd::Vel( ) ) * ts;
     }
 };
 
-int main(int argc,char * * argv)
```

Comment:
@psychocoderHPC  either you missed this, or the one in the real `main` was an error from your side (its `**` there). So what is the correct one?

Reply:
Of course, only
```C++
int main(
    int argc,
    char * argv[]
)
{
    // ...
}
```
is the right signature ;-D

Reply:
I'm fine with this too. But what about the general case of _multi pointer_ types? Are there spaces around each of them or are they packed?

Reply:
Ohh I missed that. `**` should be packed but I am also fine with `* x[]`

Reply:
What about `const` than: `int * const * foo;`?

Reply:
I never saw a const in the signature of the main function. I would not add `const`.
But if const is needed `int * const * foo` would fit

Reply:
My question was not limited to `main`!

Reply:
:ok_hand: